src/components/Match/MatchStory.jsx

```diff
@@ -152,6 +140,29 @@ class StoryEvent {
   }
 }
 
+class IntroEvent extends StoryEvent {
+  constructor(match) {
+    super(-90);
+    this.game_mode = match.game_mode;
+    this.region = match.region;
+    this.date = new Date(match.start_time * 1000);
+  }
+  get localizedRegion() {
```

Comment:
By updating the updateconstants file to write the prettified text string.  Putting strings in the language files is another option, something like `region_1`, etc.